GraphQL.md

```diff
@@ -19,47 +19,74 @@ POST https://yourdomain.com/graphql
 The structure for our GraphQL payloads is generated and, therefore, uniform across all data types. To understand our format, it is strongly recommended that you first have a [basic understanding of GraphQL](http://graphql.org/learn/). The general structure is as follows:
 
 ```
-rootObjectTypeName([op: FETCH|ADD|REPLACE|DELETE, id: ID-VALUE, data: DATA-OBJ]) {
-  exposedFields
+rootObjectTypeName([op: UPSERT|REPLACE|REMOVE|DELETE, ids: [ID-VALUES], data: DATA-OBJ]) {
```

Comment:
Hmm, this is obsolete. Refer to the [readme](https://github.com/yahoo/elide/blob/elide-4.x/graphql-elide.md) in `elide-4.x` branch. Although, I still need to update it w `sort`, `filter`, `first` and `offset` arguments.